### PR TITLE
Add Gradient Fill + Stroke

### PR DESCRIPTION
**Pull request by b-wils**
This is documenting the current implementation in BodyMovin and players. There are a few rough edges/shortcomings in this implementation which I tried to capture in these docs. This is something we may want to improve at a later date, though probably not in the 1.0 release given the additional work that would be required update and migrate.

### CODE REVIEW

File: docs/specs/values.md
Comment (mbasaglia) [inline docs/specs/values.md:71]: Probably we want an embedded gradient preview. If needed I can add it later since I'm familiar with the markdown extension.

File: schema/constants/gradient-type.json
Comment (mbasaglia) [inline schema/constants/gradient-type.json:9]: Could be useful to include a short description

File: schema/properties/gradient-property.json
Comment (mbasaglia) [inline schema/properties/gradient-property.json:8]: Maybe "Stop count" is a better title?

File: docs/specs/values.md
Comment (b-wils) [reply docs/specs/values.md:71]: There is an example/playground with the gradient with transparency. Did you want one for the non-transparency gradient as well or are you referring to something else with preview?

File: docs/specs/values.md
Comment (b-wils) [reply docs/specs/values.md]: This is great! Took it almost word for word.

Note- the formula formatting increased the line height and looked out of place in a paragraph, so I used code formatting instead.

File: docs/specs/shapes.md
Comment (b-wils) [reply docs/specs/shapes.md]: Deleted these from the JSON when not applicable. Do you know if it is possible to make the inputs in a lottie-playground conditional as well?

**Review by fmalita — APPROVED** [review]

**Review by jbeta51 — APPROVED** [review]

File: docs/specs/values.md
Comment (mbasaglia) [reply docs/specs/values.md:68]: I mean like the inline thing we have for colors, just to have a visual representation after the table

File: docs/specs/shapes.md
Comment (mbasaglia) [reply docs/specs/shapes.md]: > Do you know if it is possible to make the inputs in a lottie-playground conditional as well?

Not currently

File: docs/specs/values.md
Comment (b-wils) [reply docs/specs/values.md:68]: Ah got it. I see the scripts for it, but couldn't get it to work. Is there an example somewhere? Or I could leave it to you.

**Review by jbeta51 — APPROVED** [review]

File: docs/specs/values.md
Comment (mbasaglia) [reply docs/specs/values.md:68]: I can handle it, once all the current MRs are merged I can go through and make an editorial pass to make sure everything is consistent

**Review by mbasaglia — APPROVED** [review]

**Review by Aidosmf — APPROVED** [review]

**Pull request merged**
